homeassistant/components/statistics/sensor.py

```diff
@@ -485,16 +485,14 @@ def _fetch_states_from_database(self) -> list[State]:
         else:
             start_date = datetime.fromtimestamp(0, tz=dt_util.UTC)
             _LOGGER.debug("%s: retrieving all records", self.entity_id)
-        entity_states = history.state_changes_during_period(
+        return history.state_changes_during_period(
             self.hass,
             start_date,
             entity_id=lower_entity_id,
             descending=True,
             limit=self._samples_max_buffer_size,
             include_start_time_state=False,
-        )
-        # Need to cast since minimal responses is not passed in
-        return cast(list[State], entity_states.get(lower_entity_id, []))
+        ).get(lower_entity_id, [])
```

Comment:
Not a particular fan of this change. It doesn't change the behavior but was it needed to scram everything into one command? Just my opinion, no need to discuss or revert :)

Reply:
I'm not sure what to do with this feedback.

Reply:
Hey @bdraco,
nevermind. Personal preference I guess. Thanks for your work!